lighthouse-core/lib/i18n/i18n.js

```diff
@@ -473,6 +473,19 @@ function replaceIcuMessageInstanceIds(inputObject, locale) {
   return icuMessagePaths;
 }
 
+/** @typedef {import('./locales').LhlMessages} LhlMessages */
+
+/**
+ * Populate the i18n string lookup dict with locale data
+ * Used when the host environment selects the locale and serves lighthouse the intended locale file
+ * @see https://docs.google.com/document/d/1jnt3BqKB-4q3AE94UWFA0Gqspx8Sd_jivlB7gQMlmfk/edit
+ * @param {LH.Locale} locale
+ * @param {LhlMessages} lhlMessages
```

Comment:
similarly we should shape check this (a la plugins/budgets.json). All our good error handling code assumes the type is correct (and isn't such good error handling code if it isn't)